f5/bigip/resource.py

```diff
@@ -369,6 +377,48 @@ def raw(self):
         """
         return self.__dict__
 
+    def _key_dot_replace(self, rdict):
```

Comment:
As I mention in the rebased PR...  I have reservations about the DottedDict...  why are we adding that?